websites/W/Wolfy/presence.ts

```diff
@@ -2,7 +2,6 @@ const presence = new Presence({
 		clientId: "721798189111443537"
 	}),
 	strings = presence.getStrings({
-		play: "presence.playback.playing",
 		browsing: "presence.activity.browsing"
 	});
```

Comment:
Do we really need this?

Reply:
It was from old code, I removed it because `play` is translated in `lecture` like if it was "playing music",  it doesn't fit in french

Reply:
It seems pretty silly that only 1 string is used, considering that all the other ones are hardcoded.

Reply:
I will look at multi-langage and adding more strings in this to have no more hardcoded strings

Reply:
According to Guidelines, i can't create custom strings with multiLanguage until this presence have 1000 users , so i will just hardcoded all strings